container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -951,10 +968,23 @@ impl_runtime_apis! {
         }
 
         fn elasticity() -> Option<Permill> {
-            None
+            Some(pallet_base_fee::Elasticity::<Runtime>::get())
```

Comment:
This isn't needed for the upgrade, but I noticed it was set like that in the frontier template, and this compiles. So need someone to confirm whether we want this to Some or None.

Reply:
it really depends. But as long as base-fee is there you can leave it like this